qa/integration/src/test/resources/features/broker/DeviceBrokerI9n.feature

```diff
@@ -10,8 +10,9 @@
 #     Eurotech - initial API and implementation
 ###############################################################################
 @broker
-@integration
 @deviceBroker
+@integration
```

Comment:
Hey @ct-anaalbic,
you deleted the "integration" tag and after that added it 2 lines later - can you put this back? 😄

Reply:
Hi @LeoNerdoG,
yes, I will resolve this. :)